source/includes/Person/persons.md

```diff
@@ -220,10 +411,49 @@ If user not logged in to perform this action, a `401 Unauthorized` status is ret
 
 ## Delete person name subresource
 
-```console
+> Delete person name subresource
+
+```shell
 DELETE /person/:target_person_uuid/person/:target_name_uuid
 ```
 
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("text/plain");
+RequestBody body = RequestBody.create(mediaType, "");
+Request request = new Request.Builder()
+  .url("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/person/90f7f0b4-06a8-4a97-9678-e7a977f4b518/name/4e5c5c75-323e-43f4-afad-c5970209d0f9?purge=true")
```

Comment:
I saw a mismatch in the documentation. Please use the demo only here afterward for the samples.

Reply:
sure @Akayeshmantha noted